Core/MIPS/x86/Jit.cpp

```diff
@@ -274,6 +274,11 @@ void Jit::Compile(u32 em_address) {
 		ClearCache();
 	}
 
+	if (!Memory::IsValidAddress(em_address)) {
```

Comment:
Do we need this in ArmJit and Arm64Jit?

-[Unknown]

Reply:
I don't know if we need it at all. I think when using the JIT, the exception handler will always catch it in the dispatcher.

Reply:
Maybe there's some other path we end up in Compile, I dunno...